src/pages/identity/activity-boxes/SocialProofs.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import StatusIndicator from './shared/StatusIndicator';
+import GenericList from './shared/GenericList';
+import { FacebookIcon, TwitterIcon, LinkedinIcon, GithubIcon, TextButton } from '@nomios/web-uikit';
+import styles from './SocialProofs.css';
+
+class SocialProofItem extends Component {
+    render() {
+        const { network, account } = this.props;
+
+        let socialIcon = null;
+
+        switch (network) {
```

Comment:
I would change this to a new function which returns the icon.

Like so, your render method would be clearer and it would be like this: 

```js
const socialIcon = this.inferSocialIcon(network);
```

But the change it's up to you, it's a personal preference.

Reply:
No, I agree. Just changed it 😄